src/libs/ReportUtils.ts

```diff
@@ -4447,7 +4447,13 @@ function navigateBackOnDeleteTransaction(backRoute: Route | undefined, isFromRHP
         return;
     }
     if (isFromRHP) {
-        Navigation.dismissModal();
+        if (reportID) {
```

Comment:
How do we know that just because a reportID is passed that this is a tracked expense?